README.md

```diff
@@ -51,12 +51,9 @@ Questions can be asked by raising an `Issue`.
 ```
 
 - Open the file `README.md`
-
```

Comment:
There are changes on lines 54, 56, 59. I'm not sure why there are changes here. Could you explain?

Reply:
Okay cool, got your point @AllanRegush. I just noticed the changes. 

![image](https://user-images.githubusercontent.com/47639176/109762984-6aa19b00-7c17-11eb-813f-68e1881384c5.png)

After forking when I opened from VSCode it showed me this interface I just added my Name and Github Profile Url that's all.
I might mistakenly, pressed the Enter or something like that. I'm sorry from my side.
Do I need to delete this request and make a pull request again?

Reply:
It looks like a few spaces were removed